credentials/apps/verifiable_credentials/composition/open_badges.py

```diff
@@ -83,6 +94,8 @@ def get_types(cls):
 class OpenBadges301DataModel(OpenBadgesDataModel):  # pylint: disable=abstract-method
     """
     Open Badges data model.
+
+    NOTE: for example purposes, for now.
```

Comment:
Maybe we should be more specific for community, e.g., like this:

```suggestion
    Open Badges version 3.0.1 data model.

    NOTE: this is example of extending data model with custom fields and overrides.
```